content/documentation/deploy/packing.md

```diff
@@ -44,7 +44,7 @@ When the build finishes, you have a fresh baked binary in the `bin` folder. It w
 To get the list of available options, use the help command:
 
 ```bash
-$ buffalo help build
+$ buffalo build --help
```

Comment:
currently, both the `help` sub-command and the `--help` flag are valid ways to get the same help message so I don't think we need to change this. (also, I found that using flags are slightly harder in some situation so keeping the `help` as the main method to get a help message could be better in my opinion.)

One possible issue with consistency is when using plugins. If we also consider that consistency, keeping the `--help` flag but deprecating the `help` command could be one possible option. @paganotoni, I think we need a decision here. (whether using the `help` command as the preferred method or just using `--help` and deprecating the `help` command.)

In that case, we also need to change the line above:
* from: "To get the list of available options, use the help command:"
* to: "To get the list of available options, use the `--help` flag:

Reply:
Ok, thanks for the feedback, I'm going to add something else to this ,  `buffalo db help` is diferent from `buffalo db --help`.

- `buffalo help db`: 

[PLUGIN] A tasty treat for all your database needs

Usage:
  buffalo pop [flags]

Aliases:
  pop, db

Flags:
  -h, --help   help for pop 

- buffalo db --help: 

A tasty treat for all your database needs

Usage:
  buffalo-pop pop [flags]
  buffalo-pop pop [command]

Aliases:
  pop, db

Available Commands:
  create      Creates databases for you
  destroy     Allows to destroy generated code.
  drop        Drops databases for you
  fix         Brings pop, soda, and fizz files in line with the latest APIs
  generate    Generates config, model, and migrations files.
  migrate     Runs migrations against your database.
  reset       Drop, then recreate databases
  schema      Tools for working with your database schema

Flags:
  -c, --config string   The configuration file you would like to use.
  -d, --debug           Use debug/verbose mode
  -e, --env string      The environment you want to run migrations against. Will use $GO_ENV if set. (default "development")
  -h, --help            help for pop
  -p, --path string     Path to the migrations folder (default "./migrations")
  -v, --version         Show version information

Use "buffalo-pop pop [command] --help" for more information about a command.

- All other commands are the same for `help` and `--help` except db and pop.

Reply:
Yes, exactly. That is what I said as:

> One possible issue with consistency is when using plugins. <...>

Since the execution of `buffalo help pop` is directly handled by `buffalo` and it only puts short basic messages, but the execution `buffalo pop --help` is done by the plugin itself so it can do print much informational messages, which is good thing.

So for plugins, flag method is much better while the others are just a style difference. From this perspective, I agree to change the default as flag and deprecation the command method to reduce confusions. It is decision issue basically. I would vote to flag method :-)

(Actually, this is one of my ongoing agenda to think more since I found the issue with the parameter parsing.)